### PR TITLE
add countdown component to game page

### PR DESCRIPTION
**Pull request by R1ZZU**

### CODE REVIEW

File: scenarios/game/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+require('./countdown');
```
Comment (KantinDmitry) [inline scenarios/game/index.js]: maybe import for consistency?

File: src/game/containers/RoundContainer.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const PureRound = ({ roundPhase, timeRemaining }) =>
+    roundPhase !== RoundPhase.IN_PROGRESS && roundPhase !== RoundPhase.FINISHED ? null :
+
+    <RoundLayout>
+        <Countdown
+            timeAmount={DURATION}
+            timeRemaining={timeRemaining}
+        />
+    </RoundLayout>
```
Comment (KantinDmitry) [inline src/game/containers/RoundContainer.js]: I see SelectorInput in src/game/components/RoundLayout.js.
Should SelectorInput be added here?

File: src/game/game-store.js [hunk 3]
```diff
@@ -2,4 +2,6 @@ import { createStore } from 'redux';
 import { gameReducer } from './reducers/game-reducer.js';
 
 const store = createStore(gameReducer);
+window.store = store;
```
Comment (KantinDmitry) [inline src/game/game-store.js]: Just curious, maybe it makes sense to add NODE_END and add the store to window conditionally?

File: scenarios/game/countdown.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { store } from '../../src/game/game-store';
+import * as RoundPhaseActions from '../../src/shared/actions/round-phase-actions';
+import * as CountdownActions from '../../src/shared/actions/countdown-actions';
+import { DURATION } from '../../src/shared/constants/round';
+
+store.dispatch(RoundPhaseActions.startRound());
+
+async function pause(ms) {
```
Comment (Shastel) [inline scenarios/game/countdown.js]: Is `async` required here?

File: src/game/containers/RoundContainer.js (hunk 2 above)
Comment (R1ZZU) [reply src/game/containers/RoundContainer.js]: I want to add it alongside with redux stuff. Layout node is just a placeholder for future PR

File: scenarios/game/countdown.js (hunk 4 above)
Comment (R1ZZU) [reply scenarios/game/countdown.js]: it is not _required_, as there is no `await` statement inside function body, however I consider this a good coding style. IMO everything that could be `await`-ed should be `async`

**Pull request merged**